voodoo-gen.opam

```diff
@@ -11,7 +11,7 @@ bug-reports: "https://github.com/jonludlam/voodoo/issues"
 depends: [
   "dune" {>= "2.8"}
   "ocaml" {>= "4.08.0"}
-  "omd" {>= "2.0.0~alpha2"}
+  "omd" {= "2.0.0~alpha2"}
```

Comment:
Any reason for not using alpha3? (it's fixing numerous issues on the generated documentation)

Reply:
The current state of voodoo is not working with `omd` alpha3, it has breaking changes. The CI is currently broken because of that so it's taking the easiest path for now.

Nevertheless having support for `omd.2.0.0~alpha3` and `odoc.2.2.0` is welcomed !

Reply:
alpha3 unfortunately seems to break the pipeline, so we should focus on fixing whatever issue the pipeline has first and then upgrade to alpha3